### PR TITLE
Parse the static sit config, inject pages url

### PR DESCRIPTION
**Pull request by smithaborkar**
WIP

CC: https://github.com/github/pages-engineering/issues/1295

### CODE REVIEW

**Comment by YiMysty** [comment]
verified work https://github.com/paper-spa/actions-pages-nuxtjs/runs/6945080976?check_suite_focus=true#step:6:1

**Review by felipesu19 — APPROVED** [review]

File: nuxt.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default {
+    router: {
+        base: '/amazing-new-repo/'
+    }
+}
```
Comment (JamesMGreene) [inline nuxt.config.js:5]: As far as I can tell, this file seems unnecessary. 🤔

Comment (YiMysty) [reply nuxt.config.js:5]: good find~ will remove it :)

File: src/set-pages-path.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      default:
+        throw "Unknown config type"
+    }
+
+    let configParser = new ConfigParser(ssConfig)
+    configParser.parse()
+
+  } catch (error) {
+    core.error('Set pages path in the static site generator config failed', error)
+    console.log(error)
```
Comment (YiMysty) [inline src/set-pages-path.js:43]:
```suggestion
    core.error('Set pages path in the static site generator config failed', error)
```

**Pull request merged**